### ISSUE TITLE
What is the prompting method used for training and testing?

### ISSUE THREAD

**Issue by thangngoc89** [hitachinsk/SAMed#2]
May I ask what is the prompting method for predicting the mask? It's unclear from the paper and the code. 
Is it point prompting, box prompting or automatic prompting using a grid of points?

Thank you very much

**Comment by hitachinsk** [comment]
Automatic prompting, but we did not use a grid of points. Instead, we finetune the default embedding adopted in the original segment anything model.

**Comment by thangngoc89** [comment]
@hitachinsk thank you for your answer. I'm looking to use your method on my datasets. My dataset is medical related RGB photos but it doesn't has any labelled dataset. I want to do automatic segmentation with the least amount of samples. Original SAM model's automatic prompting mode could is segmenting an object to many small pieces and it's very hard to merge them together using normal algorithm and rules

**Comment by thangngoc89** [comment]
@hitachinsk Thank you for your suggestions. I have annotated around 200 images using SAM (manually selecting points). I'm a little bit confused about the number of classes. There are only 2 classes of object in my dataset but a single RGB images could has 10-14 objects of the same class. From reading your code, it seems like you expected to have one object per class on every images. Do you have any suggestions on handling this situation? Thank you in advance.
